webpack/webpack.parts.js

```diff
@@ -64,7 +64,7 @@ exports.extractScss = ({mode = 'production'}) => ({
   },
   plugins: [
     new MiniCssExtractPlugin({
-      filename: 'css/[name].css',
+      filename: ({ chunk: {name} }) => name === 'rtl' ? 'css/theme.rtlfix' : 'css/[name].css',
```

Comment:
```suggestion
      filename: ({ chunk: {name} }) => name === 'rtl' ? 'css/theme.rtlfix' : 'css/[name].css',
```

Is the theme.rtlfix a real maintainable solution? What about a simple rtl folder with .scss files that we can generate, and include it if the rtl is activated? I feel like it's more maintainable, wdyt?

Reply:
I had a problem with rtl.css in public/css/rtl.css
StylsheetGenerator will create this file: rtl_rtl.css 
![Screenshot 2022-01-03 at 09-43-26 Theme Logo Theme • sidebar](https://user-images.githubusercontent.com/85633460/147916987-c6b0acb1-f85a-4108-9eb6-2311ff10a814.png)

Reply:
I'm not sure to understand